package.json

```diff
@@ -29,15 +29,15 @@
     "commitplease": "2.3.1",
     "core-js": "2.2.2",
     "cross-spawn": "2.2.3",
+    "eslint-config-jquery": "github:markelog/eslint-config-jquery",
```

Comment:
Can we get rid of that before ladning? I'd like to avoid having commits on `master` that in some time will almost for sure stop compiling, even on the original environment.